screens/WalletDetailScreen.tsx

```diff
@@ -197,6 +212,36 @@ const WalletDetailScreen = ({
       ? null
       : amountDecimal;
 
+  useEffect(() => {}, [
+    tokenBlacklist,
+    addTokenToBlackList,
+    removeTokenFromBlackList
+  ]);
+
+  const hideButton = (tokenId: string) => (
+    <ButtonGroup>
+      <Button
+        nature={"cautionGhost"}
+        onPress={() => {
+          addTokenToBlackList(tokenId);
+        }}
+        text="Hide this Token"
+      />
+    </ButtonGroup>
+  );
+
+  const showButton = (tokenId: string) => (
+    <ButtonGroup>
+      <Button
+        nature={"inverse"}
+        onPress={() => {
+          removeTokenFromBlackList(tokenId);
+        }}
+        text="Show this Token"
```

Comment:
Will probably get changed anyways when we find a better button placement, but `this` can be omitted here as we're on this specific token's page already.  
Trying to think of how the messaging for this might be improved as the user doesn't know what show/hide/blacklists are.  Perhaps something which alludes to adding/removing from the token vault, since that's a concept/name we show to the user.